common/game_concepts/wc_other_game_concepts.txt

```diff
@@ -0,0 +1,2 @@
+﻿wc_mod = {
```

Comment:
Could you add some kind of tooltip so you know what doctrine you need to change?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/108220896-a7b65980-7150-11eb-8c6c-e02d03129eb2.png)

</details>